src/main/resources/liquibase-addCvChangeLog.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+  <changeSet id="add-column-CV" author="kholod">
+    <addColumn tableName="student">
+      <column name = "cv" type = "LONGBLOB">
```

Comment:
We have another task for do it with User. I think here we must do it in same way. So, I propose to realise this together with User's image, maybe in next Sprint.